src/Common/test/Common.TestResources/TestHelpers.cs

```diff
@@ -92,7 +92,9 @@ public static IConfiguration GetConfigurationFromDictionary(IDictionary<string,
             { "postgres:client:timeout", "1" },
             { "redis:client:abortOnConnectFail", "false" },
             { "redis:client:connectTimeout", "1" },
-            { "sqlserver:credentials:timeout", "1" }
+            { "sqlserver:credentials:timeout", "1" },
+            { "management:metrics:export:wavefront:apiToken", "test" },
+            { "management:metrics:export:wavefront:uri", "http://test.io" },
```

Comment:
This dictionary was originally intended just to make sure tests run quickly (or in the case of the Eureka Service URL, pass in the CI environment) - what would happen if these two new settings weren't included?

Reply:
The wavefront test needs it.